packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -774,20 +775,27 @@ void WebView::InitWebView() {
       [this]() -> LWE::WebContainer::ExternalImageInfo {
         std::lock_guard<std::mutex> lock(mutex_);
         LWE::WebContainer::ExternalImageInfo result;
-        if (!candidate_surface_) {
-          candidate_surface_ = tbm_pool_->GetAvailableBuffer();
+        if (!working_surface_) {
+          working_surface_ = tbm_pool_->GetAvailableBuffer();
         }
-        if (candidate_surface_) {
-          result.imageAddress =
-              static_cast<void*>(candidate_surface_->Surface());
+        if (working_surface_) {
+          result.imageAddress = static_cast<void*>(working_surface_->Surface());
         } else {
           result.imageAddress = nullptr;
         }
         return result;
       },
       [this](LWE::WebContainer* c, bool isRendered) {
         if (isRendered) {
-          texture_registrar_->MarkTextureFrameAvailable(GetTextureId());
+          std::lock_guard<std::mutex> lock(mutex_);
+          if (candidate_surface_) {
+            tbm_pool_->Release(candidate_surface_);
+            candidate_surface_ = nullptr;
+          } else {
+            texture_registrar_->MarkTextureFrameAvailable(GetTextureId());
+          }
+          candidate_surface_ = working_surface_;
```

Comment:
Q. It looks like `working_surface_` could be `nullptr` because `tbm_pool_->GetAvailableBuffer() `can return `nullptr`. nevertheless, is its safety guaranteed?

Reply:
No. But I don't assume such a situation If we met that situation It should fail Immediately.
This is intended thing.